components/steps/Profile.tsx

```diff
@@ -291,6 +291,7 @@ const ProfileStep: React.FunctionComponent<Props> = props => {
                     value="american-indian"
                     defaultChecked={profile.ethnicity === "american-indian"}
                     disabled={submitted}
+                    required
```

Comment:
til: if you add `required` to a radio button, it affects the entire radio group (input[radio] w/ the same name)